lib/src/base-styles/_nx-card.scss

```diff
@@ -107,75 +78,64 @@
   }
 
   .nx-card__content {
-    -ms-grid-row: 2;
-    -ms-grid-column: 2;
-    grid-area: card-text;
+    align-self: center;
+    grid-area: card-content;
     margin: 0;
     padding: 0;
+    row-gap: 0;
   }
 
   .nx-card__call-out {
-    -ms-grid-row-align: center;
     align-self: center;
```

Comment:
This is redundant with the `height: 100%`.  One or the other can be removed

Reply:
> This is redundant with the `height: 100%`. One or the other can be removed

Here and below `height: 100%` is necessary to get the border to display full height.

Reply:
> > This is redundant with the `height: 100%`. One or the other can be removed
> 
> Here and below `height: 100%` is necessary to get the border to display full height.

However changing the cards to flexbox solved that: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/1a904c393841bca80421915a60e0b8e1d7f4305f